src/components/ScanBoxLoadingMessage.js

```diff
@@ -1,15 +1,29 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { Row, Col } from 'react-flexbox-grid';
 import '../assets/css/ScanBox.css';
 
-const ScanBoxLoadingMessage = () => (
-  <Row >
+const ScanBoxLoadingMessage = ({ longLoading }) => {
+  const caption = longLoading ?
+    'ICO Scanning in process, please wait this one is big it will take a while...'
```

Comment:
Interpunction missing. Maybe: 
`ICO Scanning in process, please wait. This one is big, it will take a while...`